src/Sylius/Bundle/AdminBundle/Resources/views/Product/Update/_breadcrumb.html.twig

```diff
@@ -0,0 +1,11 @@
+{% import '@SyliusAdmin/Macro/breadcrumb.html.twig' as breadcrumb %}
+
+{% set breadcrumbs = [
+        { label: 'sylius.ui.administration'|trans, url: path('sylius_admin_dashboard') },
+        { label: 'sylius.ui.products'|trans , url: path('sylius_admin_product_index') },
+        { label: resource.name|trans },
```

Comment:
It doesn't work this way, name that is returned from resource is already translated